docs/user-guide/how-to.md

```diff
@@ -366,7 +366,15 @@ can use any of the following annotations:
 
 * **`propagate.hnc.x-k8s.io/none`**: Setting `none` to `true` (case insensitive)
   will result in the object not propagating to _any_ descendant namespace. Any
-  other value will be rejected.
+  other value will be rejected. The behavior would be the same as setting
+  [synchronization mode](#modify-the-resources-propagated-by-hnc) to 
+  `AllowPropagate` with no selectors at all.
```

Comment:
I think this is a bit confusing - maybe better to say something like "This is only useful in the `Propagate` mode." Ditto in the next section, I might just say "This is only useful in the `AllowPropagate` mode."

Reply:
Yes, I was wondering how to phrase it, this sounds better